Src/FluentAssertions/CallerIdentification/HandlerResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace FluentAssertions.CallerIdentification
+{
+    public enum HandlerResult
```

Comment:
🤔 Isn't this the _state_ of the parsing process? If so, I think renaming this to `ParsingState` or something like that makes the algorithm a lot easier to grasp.

Reply:
Done